beacon-chain/powchain/service.go

```diff
@@ -721,7 +721,7 @@ func (s *Service) logTillChainStart() {
 	}
 
 	log.WithFields(logrus.Fields{
-		"Extra validators needed":   valNeeded,
-		"Time till minimum genesis": fmt.Sprintf("%s", time.Duration(secondsLeft)*time.Second),
+		"Extra validators needed":  valNeeded,
+		"Time till eth1 threshold": fmt.Sprintf("%s", time.Duration(secondsLeft)*time.Second),
```

Comment:
Eth1 threshold is not a time. I don't quite understand this

Reply:
There are two countdowns, and the first countdown with 'minimum genesis' was confusing for users(because of MIN_GENESIS_TIME).